pkg/policies/opa/rego/aws/aws_instance/userDataWithSecrets.rego

```diff
@@ -0,0 +1,12 @@
+package accurics
+
+
+EC2withSecrets[retVal] {
+    pattern := ["[A-Za-z0-9/+=]{40}","(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}","(\"|')?(AWS|aws|Aws)?_?(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)(\"|')?\\s*(:|=>|=)\\s*(\"|')?[A-Za-z0-9/\\+=]{40}(\"|')?"]
```

Comment:
Can you double check the pattern? I'm getting an error message while testing [here](https://play.openpolicyagent.org/p/bZG4Veoi2H)

Reply:
@cesar-rodriguez, there might be some mistake while I copied it from my Visual Code editor:
please try this pattern: ["[A-Za-z0-9/+=]{40}","(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}","(\"|')?(AWS|aws|Aws)?_?(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)(\"|')?\\s*(:|=>|=)\\s*(\"|')?[A-Za-z0-9/\\+=]{40}(\"|')?"]

Reply:
One more suggestion--is there a working way to do case-insensitive matches in rego's regex implementation? Otherwise, I think something like ([Aa][Ww][Ss]) is preferable to (AWS|aws|Aws).

Reply:
@harkirat22 can we expect an update on this PR, or should we close it?

Reply:
@jlk on it. Will update it today